<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admin-PC" ignored="0" name="nEXBOX_TestCase1" tests="4" failures="2" timestamp="06 Jan 2020 21:46:08 GMT" time="57.809" errors="0">
  <testcase name="Url_Hit" time="25.465" classname="com.TestCase.Login_TestCase_Folder.Login_TestCase"/>
  <testcase name="Without_Password" time="16.153" classname="com.TestCase.Login_TestCase_Folder.Login_TestCase">
    <failure type="java.lang.AssertionError" message="WithOut Password Successfully login...== expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: WithOut Password Successfully login...== expected [true] but found [false]
at com.PageObject.Loginpage.LoginPage.UserName(LoginPage.java:45)
at com.TestCase.Login_TestCase_Folder.Login_TestCase.Without_Password(Login_TestCase.java:15)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Without_Password -->
  <testcase name="Without_UserName" time="9.332" classname="com.TestCase.Login_TestCase_Folder.Login_TestCase">
    <failure type="java.lang.AssertionError" message="WithOut UserName Successfully login...== expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: WithOut UserName Successfully login...== expected [true] but found [false]
at com.PageObject.Loginpage.LoginPage.Password_Data(LoginPage.java:70)
at com.TestCase.Login_TestCase_Folder.Login_TestCase.Without_UserName(Login_TestCase.java:19)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Without_UserName -->
  <testcase name="With_Username_Password" time="6.689" classname="com.TestCase.Login_TestCase_Folder.Login_TestCase"/>
</testsuite> <!-- nEXBOX_TestCase1 -->
